### PR TITLE
Replace all occurences of `mix phoenix` with `mix phx`

### PR DESCRIPTION
**Pull request by mhutter**
At least where it makes sense. Yay for less confusion and more copy/pasting!

### CODE REVIEW

File: lib/mix/tasks/phoenix.routes.ex [hunk 1]
```diff
@@ -4,8 +4,8 @@ defmodule Mix.Tasks.Phoenix.Routes do
   @moduledoc """
   Prints all routes for the default or a given router.
 
-      $ mix phoenix.routes
-      $ mix phoenix.routes MyApp.AnotherRouter
+      $ mix phx.routes
+      $ mix phx.routes MyApp.AnotherRouter
```
Comment (chrismccord) [inline lib/mix/tasks/phoenix.routes.ex]: this is the deprecated task so these changes should be reverted

File: installer/templates/new/config/prod.exs [hunk 2]
```diff
@@ -9,7 +9,7 @@ use Mix.Config
 #
 # Finally, we also include the path to a cache manifest
 # containing the digested version of static files. This
-# manifest is generated by the mix phoenix.digest task
+# manifest is generated by the mix phx.digest task
```
Comment (chrismccord) [inline installer/templates/new/config/prod.exs]: this is the deprecated template, so these changes should be reverted

File: installer/templates/new/README.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ To start your Phoenix app:
   * Install dependencies with `mix deps.get`<%= if ecto do %>
   * Create and migrate your database with `mix ecto.create && mix ecto.migrate`<% end %><%= if brunch do %>
   * Install Node.js dependencies with `npm install`<% end %>
-  * Start Phoenix endpoint with `mix phoenix.server`
+  * Start Phoenix endpoint with `mix phx.server`
```
Comment (chrismccord) [inline installer/templates/new/README.md]: This is the deprecated template, so these change should be reverted

**Comment by mhutter** [comment]
Thanks for the feedback, I have updated the PR accordingly

**Comment by chrismccord** [comment]
❤️❤️❤️🐥🔥

**Pull request merged**